### ISSUE TITLE
[Test] Login 

### ISSUE THREAD

**Issue by mikkang** [KAIST-IS521/TeamFive#13]
#9  테스트

**Comment by bjgwak** [comment]
1. login success case (right ID & Password)
2. login failure case (wrong ID & Password)

**Comment by bjgwak** [comment]
libcurl로 현재 서버에 로그인하려고 할 때 csrf 토큰이 없어서 post를 못함. 매뉴얼하게 넣어주는 코드 필요?

**Comment by JeongOhKye** [comment]
csrf 토큰값을 읽어온다음 그 값으로 로그인 진행하도록 하면 될거 같네요

**Comment by blukat29** [comment]
네. https://github.com/KAIST-IS521/TeamFive/blob/master/web/auto_login.py 이 부분 참고해서 csrf_token값을 파싱해서 넣어줘야 할 거예요.
